docs/pages/access-graph/getting-started-on-prem.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: Getting started with Teleport Access Graph (TAG)
+---
+
+<Notice type="note">
```

Comment:
No need for tags here. I would also add a paragraph here that summarizes the scope of this page (what would a user hope to learn by following it?).